cg/meta/workflow/mip.py

```diff
@@ -308,18 +309,15 @@ def get_pipeline_version(self, case_id: str) -> str:
         sample_info: MipBaseSampleInfo = parse_sample_info(sample_info_raw)
         return sample_info.mip_version
 
-    def is_analysis_finished(self, sample_info_path: Path) -> bool:
-        """Return True if analysis is finished"""
-        sample_info_raw = yaml.safe_load(sample_info_path.open())
-        sample_info: MipBaseSampleInfo = parse_sample_info(sample_info_raw)
-        return sample_info.is_finished
-
     def get_cases_to_store(self) -> List[models.Family]:
         """Retrieve a list of cases where analysis finished successfully,
         and is ready to be stored in Housekeeper"""
         finished_cases: List[models.Family] = []
         for case_object in self.get_running_cases():
-            if self.is_analysis_finished(self.get_sample_info_path(case_object.internal_id)):
+            if (
+                self.trailblazer_api.get_latest_analysis_status(case_id=case_object.internal_id)
```

Comment:
Suggest to move this check into trailblazer API (is_latest_analysis_completed) because this will be reused by other pipelines in the future